README.md

```diff
@@ -26,5 +26,16 @@ Repository for TDR transfer code
 
 * Start the application using `sbt run`
 
+* If css is not displaying on localhost:9000/dashboard run `npm run build`
+
+* If npm is not installed install [nvm](https://github.com/nvm-sh/nvm) in root directory.
```

Comment:
I'd move the nvm/npm installation above the previous line, so that there's a clear order:

1. Set up nvm/npm if neccessary
2. Install npm dependencies
3. Build the assets

Reply:
ordering of document changed to reflect these steps 👌